### ISSUE TITLE
Default values not being copied to Sequelize model

### ISSUE THREAD

**Issue by murbanowicz** [feathers-plus/generator-feathers-plus#26]
They are copied into e.g. mongo, but not Sequelize.

*schema.ts
```
deleted: {
      type: 'boolean',
      default: false
    }
```

*mongo.ts
```
deleted: {
        default: false,
        bsonType: "boolean"
      }
```

*sequelize.ts
```
deleted: {
      type: DataTypes.BOOLEAN
    }
```

**Comment by eddyystop** [comment]
Thanks. Should the sequelize default type be exactly what the JSON-schema default value is in *schema.ts for all JSON-schema types?

In the mean time you can merge the default value into the sequelize model at `// !code: moduleExports // !end`

**Comment by eddyystop** [comment]
The JSON-schema `default` value is copied to the Sequelize model as-is. This may not be a complete solution as some values may need a translation. I'll handle those as information is provided me.

Published as f+/cli v0.4.20 using generator v0.2.26.

Feel free to comment or re-open.

**Comment by eddyystop** [comment]
Published as f+/cli v0.4.21 using generator v0.2.27.
